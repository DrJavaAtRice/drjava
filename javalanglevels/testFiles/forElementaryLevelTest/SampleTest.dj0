import junit.framework.TestCase;
/** Sampe TestCase class. */
class SampleTest extends TestCase {
  
  /** Tests the insert method. */
  void testJUnitLibraryCalls() {
    
    assertEquals("String test", "foo", "foo");
    assertEquals("int test", 5, 5);
    assertTrue("true test", true);
  }
}
