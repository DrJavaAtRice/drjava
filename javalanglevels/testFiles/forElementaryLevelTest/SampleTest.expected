// Language Level Converter line number map: dj*->java. Entries: 12
//     1->4         2->5         3->6         4->7         5->8         6->9         7->10        8->11   
//     9->12       10->13       11->14       12->15   
import junit.framework.TestCase;
/** Sampe TestCase class. */
public class SampleTest extends TestCase {
  
  /** Tests the insert method. */
  public void testJUnitLibraryCalls() {
    
    assertEquals("String test", "foo", "foo");
    assertEquals("int test", 5, 5);
    assertTrue("true test", true);
  }

  /** This method is automatically generated by the Language Level Converter. */
  public SampleTest() {
    super();
  }

  /** This method is automatically generated by the Language Level Converter. */
  public java.lang.String toString() {
    return getClass().getName() + "(" + 
        ")";
  }

  /** This method is automatically generated by the Language Level Converter. */
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    else if ((o == null) || (! o.getClass().equals(getClass()))) {
      return false;
    }
    else {
      SampleTest cast = ((SampleTest) o);
      return true;
    }
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int hashCode() {
    return getClass().hashCode();
  }
}
